src/mesh/NineSlicePlane.js

```diff
@@ -0,0 +1,240 @@
+var Plane = require('./Plane');
+
+/**
+ * The NineSlicePlane allows you to stretch a texture using 9-slice scaling. The corners will remain unscaled (useful
+ * for buttons with rounded corners for example) and the other areas will be scaler horizontally and or vertically
+ *  *
+ *```js
+ * var Plane9 = new PIXI.NineSlicePlane(PIXI.Texture.fromImage("BoxWithRoundedCorners.png"), 15, 15, 15, 15);
+ *  ```
+ * <pre>
+ *      A                          B
+ *    +---+----------------------+---+
+ *  C | 1 |          2           | 3 |
+ *    +---+----------------------+---+
+ *    |   |                      |   |
+ *  D | 4 |          5           | 6 |
+ *    |   |                      |   |
+ *    +---+----------------------+---+
+ *  E | 7 |          8           | 9 |
+ *    +---+----------------------+---+
+
+ *  When changing this objects width and/or height:
+ *     areas 1 3 7 and 8 will remain unscaled.
+ *     areas 2 and 8 will be stretched horizontally
+ *     areas 4 and 6 will be stretched vertically
+ *     area 5 will be stretched both horizontally and vertically
+ * </pre>
+ *
+ * @class
+ * @extends PIXI.mesh.Plane
+ * @memberof PIXI.mesh
+ * @param {PIXI.Texture} texture - The texture to use on the NineSlicePlane.
+ * @param {int} [leftWidth=10] size of the left vertical bar (A)
+ * @param {int} [topHeight=10] size of the top horizontal bar (B)
+ * @param {int} [rightWidth=10] size of the right vertical bar (C)
+ * @param {int} [bottomHeight=10] size of the bottom horizontal bar (D)
+ *
+ */
+function NineSlicePlane(texture, leftWidth, topHeight, rightWidth, bottomHeight)
+{
+	Plane.call(this, texture,4,4);
+	var DEFAULT_BORDER_SIZE= 10;
```

Comment:
It could, but that would not really be user friendly.
you'd have to calculate the widths that you want yourself before entering,

var slicer = new PIXI.mesh.NineSlicePlane(myTex, new PIXI.Rectangle(10,20, myTex.width - 10 - actualSizeOfRightBorder, myTex.height - 20 - actualSizeOfBottomRow));


also nitpicky minimally slower and less memory efficient too :package: